### ISSUE TITLE
Service bus attributes don't default to AzureWebJobsServiceBus setting

### ISSUE THREAD

**Issue by JonCanning** [Azure/azure-webjobs-sdk#1874]
Please provide a succinct description of the issue.

Latest release requires `Connection` to be specified in the attributes

Provide the steps required to reproduce the problem

Create a ServiceBus trigger without a `Connection` property

#### Expected behavior

It defaults to the `AzureWebJobsServiceBus` setting and works

#### Actual behavior

The function runtime is unable to start

#### Known workarounds

Add `Connection = "AzureWebJobsServiceBus"`

#### Related information 

Runtime 2.0.12050.0 
Microsoft.Azure.WebJobs.ServiceBus 3.0.0-beta8

**Comment by ielcoro** [comment]
Yep, I forgot to open an issue, I experienced that too, we needed to change all of our functions to use a named Connection now.
